### PR TITLE
Add release notes for July 9 release.

### PR DESCRIPTION
**Pull request by taeold**

### CODE REVIEW

File: firestore-bigquery-export/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+## Version 0.1.6
+
+fixed - Record document id in schema view and revert timestamp change (#357)
```
Comment (kevinthecheung) [inline firestore-bigquery-export/CHANGELOG.md:3]: fixed - Fixed issue with timestamp values not showing up in the `latest` view (#357)

File: firestore-bigquery-export/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
...
+feature - Record document id of changes tracked by firestore-bigquery-change-tracker package (#374)
+feature - Add document id column in changelog table and snapshot view (#376)
```
Comment (kevinthecheung) [inline firestore-bigquery-export/CHANGELOG.md:5]: feature - Add document ID column to changelog table and snapshot view (#376)

File: firestore-bigquery-export/CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,9 @@
+## Version 0.1.6
+
+fixed - Record document id in schema view and revert timestamp change (#357)
+feature - Record document id of changes tracked by firestore-bigquery-change-tracker package (#374)
```
Comment (kevinthecheung) [inline firestore-bigquery-export/CHANGELOG.md:4]: feature - Record document ID of changes tracked by firestore-bigquery-change-tracker package (#374)

File: firestore-bigquery-export/CHANGELOG.md (hunk 1 above)
Comment (taeold) [reply firestore-bigquery-export/CHANGELOG.md]: Done.

File: firestore-bigquery-export/CHANGELOG.md (hunk 3 above)
Comment (taeold) [reply firestore-bigquery-export/CHANGELOG.md]: Done.

File: firestore-bigquery-export/CHANGELOG.md (hunk 2 above)
Comment (taeold) [reply firestore-bigquery-export/CHANGELOG.md]: Done.

**Pull request merged**